### PR TITLE
beef up agentlib/javaagent config documentation

### PR DESCRIPTION
**Pull request by jamiejackson**

### CODE REVIEW

**Comment by jamiejackson** [comment]
Attempt to clarify agent configuration.

File: readme.md [hunk 1]
```diff
@@ -41,17 +41,20 @@ Add the following to your java invocation. (Tomcat users can use the `setenv.sh`
...
-
-If Lucee is running in a docker container, the `debugHost` must be `0.0.0.0`. However, be careful not to do this on a publicly-accessible, unprotected server, as you could expose the debugger to the public (which would be a major security vulnerability).
-
-The `jarPath` argument is the absolute path to the luceedebug.jar file. Unfortunately we have to say its name twice! One tells the JVM which jar to use as a java agent, the second is an argument to the java agent about where to find the jar it will load debugging instrumentation from.
-
-(There didn't seem to be an immediately obvious way to pull the name of "the current" jar file from an agent's `premain`, but maybe it's just been overlooked. If you know let us know!)
+* `agentlib`: Configures JDWP, which is the lower level Jave debugging protocol that the luceedebug agent connects with. (Note: The VS Code debugger connects to the _luceedebug_ agent, not JDWP, so JDWP/`agentlib` usually doesn't need to be modified/customized.)
```
Comment (softwareCobbler) [inline readme.md:44]: nit: `Jave` -> `Java`

**Comment by softwareCobbler** [comment]
lgtm, it's like "how clear can we make it"

File: readme.md (hunk 1 above)
Comment (jamiejackson) [reply readme.md]: Done. Also did a bit of a punctuation/grammar scrub.

**Pull request merged**